docs/storage-policies.md

```diff
@@ -63,6 +67,7 @@ To determine which storage policy is assigned to a user, call
 `StoragePoliciesManager.GetAssignmentForTargetAsync(string entityId, string entityType = "user")`
 with the ID of the user.
 
+<!-- sample get_storage_policy_assignments -->
```

Comment:
Since this "endpoint" requires a certain query param that could someday not be required (to expose true "get all" functionality), should this be a variant?  Maybe something like `get_storage_policy_assignments target`?

Reply:
Agreed, but for now we don't support variants yet. We can revisit this later when we are going to review every endpoint's documentation in detail.